Formula/juise.rb

```diff
@@ -24,12 +24,22 @@ class Juise < Formula
   depends_on "sqlite"
 
   def install
-    system "sh ./bin/setup.sh" if build.head?
+    system "sh", "./bin/setup.sh" if build.head?
+
+    # Prevent sandbox violation
+    inreplace "configure",
+      "SLAX_EXTDIR=\"`$SLAX_CONFIG --extdir | head -1`\"",
```

Comment:
Yeah, having upstream `juise` give us a configure option is probably the best compromise.

`libslax` itself allows the extension directory to be set during configure, but when I tried to set it to `"#{HOMEBREW_PREFIX}/lib/slax-0.20.1/extensions`, that causes `libslax` installation to trigger sandbox violations as well because it installs its own extensions there, too. So the `libslax` build would need to be changed to use that dir but only actually install its own extensions in `post_install`.

This is `libslax`'s guidance: https://github.com/Juniper/libslax/blob/4f35fba631166ade66e97c8c1880375572674d5a/doc/slax.txt#L3771-L3780
```

Extension libraries should be placed in the directory named that is
either:
- the /usr/local/lib/slax/extensions directory
- the "lib/slax/extensions" directory under the "configure --prefix"
  option given at build time
- the directory specified with the "configure --with-extension-dir"
  option given at build time
- one of the directories listed in the environment variable
  SLAX_EXTDIR (colon separated)
- one of the directories provided via the --lib/-L argument to "slaxproc"
```
So it may be somewhat inconvenient for `juise` users to have to set `SLAX_EXTDIR` or pass `--lib/-L`, but that seems preferable to altering the `libslax` build system.